snappy_pipeline/workflows/homologous_recombination_deficiency/__init__.py

```diff
@@ -0,0 +1,297 @@
+# -*- coding: utf-8 -*-
+"""Implementation of the ``homologous_recombination_deficiency`` step
+
+This step allows for the computation of the scarHRD score, which is a composite of
+loss-of-heterozygocity, large-scale transitions and telomeric imbalances.
+The score is a proxy for holomogous recombination deficiencies scores produced from SNP arrays.
+The software is described in `Sztupinszki et al.<https://doi.org/10.1038/s41523-018-0066-6>`,
+but it is not part of CRAN, Bioconductor or bioconda (currently).
+The implementation also relies on obsolete version of `sequenza<https://github.com/oicr-gsi/sequenza/tree/master>` &
```

Comment:
Based on this documentation, `sequenza` is installed from github. The install/environment is using CRAN. Please adjust documentation or installing process.